client_cli/pytests/test/accounts/test_accounts_query_filters.py

```diff
@@ -0,0 +1,51 @@
+import json
+import allure
+
+from src.client_cli import iroha, iroha
+
+# using existing account to have at least one account in response
+def test_filter_by_domain(GIVEN_new_one_existence_account):
+    domain = GIVEN_new_one_existence_account.domain
+    with allure.step(
+            f'WHEN client_cli query accounts '
+            f'in the "{domain}" domain'):
+        accounts = iroha.list_filter(f'{{"Identifiable": {{"EndsWith": "@{domain}"}}}}').accounts()
+    with allure.step(
+            f'THEN Iroha should return only accounts with this domain'):
+        allure.attach(
+            json.dumps(accounts),
+            name='accounts',
+            attachment_type=allure.attachment_type.JSON)
+        assert accounts
+        for account in accounts:
+            assert account.endswith(domain)
+
+def test_filter_by_account_name(GIVEN_new_one_existence_account):
+    name = GIVEN_new_one_existence_account.name
+    with allure.step(
+            f'WHEN client_cli query accounts with name "{name}"'):
+        accounts = iroha.list_filter(f'{{"Identifiable": {{"StartsWith": "{name}@"}}}}').accounts()
+    with allure.step(
+            f'THEN Iroha should return only accounts with this name'):
+        allure.attach(
+            json.dumps(accounts),
+            name='accounts',
+            attachment_type=allure.attachment_type.JSON)
+        assert accounts
+        for account in accounts:
+            assert account.startswith(name)
+
+def test_filter_by_account_id(GIVEN_new_one_existence_account):
+    account_id = GIVEN_new_one_existence_account.name + "@" + GIVEN_new_one_existence_account.domain
+    with allure.step(
+            f'WHEN client_cli query accounts with account id "{account_id}"'):
+        accounts = iroha.list_filter(f'{{"Identifiable": {{"Is": "{account_id}"}}}}').accounts()
+    with allure.step(
+            f'THEN Iroha should return only accounts with this id'):
+        allure.attach(
+            json.dumps(accounts),
+            name='accounts',
+            attachment_type=allure.attachment_type.JSON)
+        assert accounts
+        for account in accounts:
+            assert account == account_id
```

Comment:
Newline at the end of files is mandatory for PEP8-compliant Python interpreters. Please ensure that all Python code has a newline at the end of the file.